controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -335,6 +335,7 @@ func (r *SriovNetworkNodePolicyReconciler) syncPluginDaemonObjs(ctx context.Cont
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASEVERSION")
 	data.Data["ResourcePrefix"] = os.Getenv("RESOURCE_PREFIX")
 	data.Data["ImagePullSecrets"] = GetImagePullSecrets()
+	data.Data["NodeSelectorField"] = GetDefaultNodeSelector()
```

Comment:
I think we are missing a scenario here. Consider the following steps:
a. The user installs the operator
b. the user configures SriovOperatorConfig.Spec.ConfigDaemonNodeSelector = map("x": "y")
c. the config reconciler sets the plugin DaemonSet NodeSelector to "x: y"
d. the user configures a policy
c. the sriovnetworknodepolicy reconciler sets the DaemonSet node selector back to  `GetDefaultNodeSelector()`

Am I missing something here?

Reply:
I believe you are right. sriovoperatorconfig_controller.go reacts on DeamonSet objects.